packages/functions/src/instance.ts

```diff
@@ -64,6 +66,7 @@ export function instance(_options: InstanceOptions = INSTANCE_DEFAULTS): Transfo
 			// Gather a one-to-many Mesh/Node mapping, identifying what we can instance.
 			const meshInstances = new Map<Mesh, Set<Node>>();
 			scene.traverse((node) => {
+				if (allAnimationTargetNodes.includes(node)) return;
```

Comment:
Optional, a simpler check might be looking for references to the node from animations:

```suggestion
				const isAnimated = node.listParents().some((p) => p instanceof AnimationChannel);
				if (isAnimated) return;
```

But this might conflict with my other comment, so feel free to ignore. :)